agent/bench-scripts/pbench-specjbb2005

```diff
@@ -51,9 +51,9 @@ dir=""
 function usage {
 	printf "\tThe following options are available:\n\n"
 	printf -- "\t\t-C str --config=<str>            name of the test config\n"
-	printf -- "\t\t-j str --java_opts=<str>         options passed directly to the JVM\n"
+	printf -- "\t\t-j str --java-opts=<str>         options passed directly to the JVM\n"
 	printf -- "\t\t       --nr-jvms=<int>|node      number of JVMs. if = node, number of JVMs = number of NUMA nodes\n"
-	printf -- "\t\t       --node-pinning=y/n        if y, this script will use numactl to pin JVM to a node (1 per node round-robin)\n"
+	##TODO: printf -- "\t\t       --node-pinning=y/n        if y, this script will use numactl to pin JVM to a node (1 per node round-robin)\n"
```

Comment:
Can you remove this line entirely, and then just file an issue to track adding `--node-pinning` support to `pbench-specjbb2005` instead?

Reply:
Sounds good, I opened issue #1479